src/driver/sqlite-abstract/AbstractSqliteDriver.ts

```diff
@@ -288,6 +290,10 @@ export abstract class AbstractSqliteDriver implements Driver {
 
         } else if (columnMetadata.type === "simple-json") {
             value = DateUtils.stringToSimpleJson(value);
+
+        } else if ( columnMetadata.type === "simple-enum" ) {
```

Comment:
since sqlite supports `enum` now, it should have `enum` type as well

Reply:
also, don't forget to list new types in the docs (website)

Reply:
or using CHECK constraint in sqlite can't be called `enum` ?

Reply:
I cannot find native enum support on the SQLite website: https://www.sqlite.org/datatype3.html.

I would say it makes sense to have `enum` for native enum types, and `simple-enum` for the emulated enum type. That way it will be more clear when you are using a derived/emulated type.

Reply:
There is currently no section on either enums or `simple-` types on the website. Do you want me to create one in a pull request?

Also, are you aware that the website certificate is invalid when accessing it from https://typeorm.io/?

Reply:
> Also, are you aware that the website certificate is invalid when accessing it from https://typeorm.io/?

yes =(

> There is currently no section on either enums or `simple-` types on the website. Do you want me to create one in a pull request?

yes feel free please

Reply:
2 other `simple-` types are now mentioned, but `simple-enum` is not:
https://typeorm.io/entities#simple-array-column-type
https://github.com/typeorm/typeorm/issues/10077